packages/components/src/Filters/FilterDropdown.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { Dropdown, DropdownToggle, Level, DropdownProps } from '@patternfly/react-core';
+import FilterInput from './FilterInput.js';
+import './filter-dropdown.scss';
+
+export interface FilterDropdownProps extends Omit<DropdownProps, 'onSelect'> {
+  addFilter: (filterName: string | number, selectedValue: string | number, type: string) => void;
+  removeFilter: (filterName: string | number, selectedValue: string | number) => void;
+  hideCategories: [];
+  filters: object;
```

Comment:
This should use the `FilterObject` props type.

Reply:
I actually got this wrong, I think filters should be array, but what is `FilterObject`?

Reply:
Sorry I mean `FilterInputProps` that is where the prop is used.